recipe/meta.yaml

```diff
@@ -20,13 +20,15 @@ requirements:
     - ninja
     - python
   host:
+    - openssl
+  run:
 
 test:
   commands:
     - spectest-interp --help
     - wasm-interp --help
     - wasm-objdump --help
-    - wasm-opcodecnt --help
+    - wasm-stats --help
```

Comment:
`wasm-opcodecnt` was renamed to `wasm-stats` in PR ( https://github.com/WebAssembly/wabt/pull/2298 ) as part of `wasm` version `1.0.34`